OMEdit/OMEditLIB/Options/OptionsDialog.cpp

```diff
@@ -4951,6 +4954,30 @@ FMIPage::FMIPage(OptionsDialog *pOptionsDialog)
                                FMIPage::FMU_FULL_CLASS_NAME_DOTS_PLACEHOLDER + tr(" i.e.,") + " Modelica.Electrical.Analog.Examples.ChuaCircuit\n" +
                                FMIPage::FMU_FULL_CLASS_NAME_UNDERSCORES_PLACEHOLDER + tr(" i.e.,") + " Modelica_Electrical_Analog_Examples_ChuaCircuit\n" +
                                FMIPage::FMU_SHORT_CLASS_NAME_PLACEHOLDER + tr(" i.e.,") + " ChuaCircuit");
+  // platforms
+  mpPlatformsGroupBox = new QGroupBox(tr("Platforms"));
+  Label *pPlatformNoteLabel = new Label(tr("Note: The list of platforms is created by searching for programs in the PATH matching pattern \"*-*-*-*cc\".\n"
+                                           "In order to run docker platforms add docker to PATH.\n"
+                                           "A source-code only FMU is generated if no platform is selected."));
+  // set the type groupbox layout
+  QVBoxLayout *pPlatformsLayout = new QVBoxLayout;
+  pPlatformsLayout->setAlignment(Qt::AlignTop | Qt::AlignLeft);
+  pPlatformsLayout->addWidget(pPlatformNoteLabel);
+  QCheckBox *pNativePlatformCheckBox = new QCheckBox("Native");
+  pNativePlatformCheckBox->setChecked(true);
+  pNativePlatformCheckBox->setProperty(Helper::fmuPlatformNamePropertyId, "static");
+  pPlatformsLayout->addWidget(pNativePlatformCheckBox);
+  // docker platforms
+  QStringList dockerPlarforms;
+  dockerPlarforms << "x86_64-linux-gnu docker run docker.openmodelica.org/build-deps:v1.13"
+                  << "i686-linux-gnu docker run docker.openmodelica.org/build-deps:v1.13-i386"
+                  << "x86_64-w64-mingw32 docker run docker.openmodelica.org/msyscross-omsimulator:v2.0"
+                  << "i686-w64-mingw32 docker run docker.openmodelica.org/msyscross-omsimulator:v2.0";
```

Comment:
Why is the arm platform missing? This is arm32 (our image)/arm64 (multiarch/crossbuild from docker hub):
```
arm-linux-gnueabihf docker run docker.openmodelica.org/armcross-omsimulator:v2.0
aarch64-linux-gnu docker run multiarch/crossbuild
```
In order to speed up docker responsiveness, you could add `docker run --pull=never` to not wait for 10 minutes while docker downloads the container.
I would suggest:
```c++
<< "x86_64-linux-gnu docker run --pull=never multiarch/crossbuild"
<< "i686-linux-gnu docker run --pull=never multiarch/crossbuild"
<< "x86_64-w64-mingw32 docker run --pull=never multiarch/crossbuild"
<< "i686-w64-mingw32 docker run --pull=never multiarch/crossbuild"
<< "arm-linux-gnueabihf docker run --pull=never multiarch/crossbuild"
<< "aarch64-linux-gnu docker run --pull=never multiarch/crossbuild"
```
We could have both options (that we test) and the more up to date multiarch as well